examples/mix/input.json

```diff
@@ -0,0 +1,7957 @@
+{
+    "_comment": " model parameters",
+    "model": {
+	"type_map":	["Mg", "Al","Cu","Li", "Si", "Ge", "Sn", "P", "S","H","O","Hf"],
+	"type_embedding":{
+	    "neuron":		[2, 4, 8],
+	    "resnet_dt":	false,
+	    "seed":		1
+	},	    
+	"descriptor" :{
+	    "type":		"se_e2_a",
+	    "sel":		[600, 600,600,50,20,20,20,20,50,92,46,60],
+	    "rcut_smth":	2.0,
+	    "rcut":		9.00,
+	    "neuron":		[25, 50, 100],
+	    "resnet_dt":	false,
+	    "axis_neuron":	16,
+	    "type_one_side":	true,
+	    "seed":		1,
+	    "_comment":		" that's all"
+	},
+	"fitting_net" : {
+		"battery":{
+			"neuron":		[240, 240, 240],
+	    "resnet_dt":	true,
+	    "seed":		1,
+	    "_comment":		" that's all"
+		},
+		"AlCuMg":{
+			"neuron":		[240, 240, 240],
+	    "resnet_dt":	true,
+	    "seed":		1,
+	    "_comment":		" that's all"
+		},
+		"HfO2":{
+			"neuron":		[240, 240, 240],
+	    "resnet_dt":	true,
+	    "seed":		1,
+	    "_comment":		" that's all"
+		},
+		"water":{
+			"neuron":		[120, 120, 120],
+	    "resnet_dt":	true,
+	    "seed":		1,
+	    "_comment":		" that's all"
+		}
+	},
+	"_comment":	" that's all"
+    },
+
+    "learning_rate" :{
+		"battery":{
+			"type":		"exp",
+			"decay_steps":	5000,
+			"start_lr":	0.001,	
+			"stop_lr":	3.51e-8,
+			"_comment":	"that's all"
+		},
+		"AlCuMg":{
+			"type":		"exp",
+			"decay_steps":	5000,
+			"start_lr":	0.001,	
+			"stop_lr":	3.51e-8,
+			"_comment":	"that's all"
+		},
+		"HfO2":{
+			"type":		"exp",
+			"decay_steps":	5000,
+			"start_lr":	0.001,	
+			"stop_lr":	3.51e-8,
+			"_comment":	"that's all"
+		},
+		"water":{
+			"type":		"exp",
+			"decay_steps":	5000,
+			"start_lr":	0.001,	
+			"stop_lr":	3.51e-8,
+			"_comment":	"that's all"
+		}
+    },
+
+    "loss" :{
+		"battery":{
+			"type":		"ener",
+			"start_pref_e":	0.02,
+			"limit_pref_e":	1,
+			"start_pref_f":	1000,
+			"limit_pref_f":	1,
+			"start_pref_v":	0,
+			"limit_pref_v":	0,
+			"_comment":	" that's all"
+		},
+		"AlCuMg":{
+			"type":		"ener",
+			"start_pref_e":	0.02,
+			"limit_pref_e":	1,
+			"start_pref_f":	1000,
+			"limit_pref_f":	1,
+			"start_pref_v":	0,
+			"limit_pref_v":	0,
+			"_comment":	" that's all"
+		},
+		"HfO2":{
+			"type":		"ener",
+			"start_pref_e":	0.02,
+			"limit_pref_e":	1,
+			"start_pref_f":	1000,
+			"limit_pref_f":	1,
+			"start_pref_v":	0,
+			"limit_pref_v":	0,
+			"_comment":	" that's all"
+		},
+		"water":{
+			"type":		"ener",
+			"start_pref_e":	0.02,
+			"limit_pref_e":	1,
+			"start_pref_f":	1000,
+			"limit_pref_f":	1,
+			"start_pref_v":	0,
+			"limit_pref_v":	0,
+			"_comment":	" that's all"
+		}
+	
+    },
+
+    "training" : {
+	"training_data": {
+	    "systems":		{
```

Comment:
We'd better provide sampled (or faked data) into repository. Then each example can be executed by user after this repository is cloned to local.